recipes-kernel/linux/linux-nilrt-next_git.bb

```diff
@@ -7,4 +7,4 @@ require linux-nilrt-alternate.inc
 
 # This is the place to overwrite the source AUTOREV from linux-nilrt.inc, if
 # the kernel recipe requires a particular ref.
-#SRCREV = ""
+SRCREV = "ebe990bf131e45cd1523d8b18406e5b49ff61bea"
```

Comment:
The branch for the 'next' kernel in your other patch points to nilrt/22.8/5.15 which is correct for sumo but this hash is from the top of nilrt/22.8/5.19 (i.e. hardknott).